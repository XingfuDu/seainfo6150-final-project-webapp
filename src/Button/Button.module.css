.Button {
	width: 150px;
	border: none;
	color: white;
	outline: none;
	cursor: pointer;
	font: inherit;
	padding: 13px;
	padding-left: 13px !important;
	margin: 10px;
	font-weight: bold;
	border-radius: 5px;
	text-align: center;
	font-family: 'Lora', serif;
}

.Button:first-of-type {
	margin-left: 0;
	padding-left: 0;
}

.Success {
	color:white;
	background-color: #ce5e42;
}

.Success:hover {
	color:white;
	background-color: #f8704e;
}

.Danger {
	color: #944317;
	background-color: white;
}

.Danger:hover {
	background-color: #fcd4cb;
}

.Button:disabled {
	color: #ccc;
	cursor: not-allowed;
}
