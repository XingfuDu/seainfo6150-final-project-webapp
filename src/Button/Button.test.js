import React from 'react';
import Button from './Button';
import { render } from "@testing-library/react";

describe('Button tests', () => {
    function onClick() {
	}
	it('renders correctly', () => {
		const { container } = render(<Button btnType="Success" clicked={onClick}></Button>);
		expect(container).toMatchSnapshot();
	});
});
