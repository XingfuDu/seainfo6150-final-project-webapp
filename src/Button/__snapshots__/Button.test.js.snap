// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button tests renders correctly 1`] = `
<div>
  <button
    class="Button Success"
  />
</div>
`;
